docs/5.x.yaml

```diff
@@ -32,7 +32,7 @@ pages:
     - Cluster Monitoring: monitoring.md
     - Remote Management: manage.md
     - Ops Center: opscenter.md
-    - Application Catalog: catalog.md
+    - hidden: catalog.md
```

Comment:
Why has this become hidden?

Reply:
I didn't make this change. Perhaps @kontsevoy knows.